rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -294,15 +295,14 @@ pub(super) fn unspecified_err(_e: aws_lc_rs::error::Unspecified) -> Error {
 
 #[cfg(test)]
 mod tests {
-    #[cfg(feature = "fips")]
-    #[test]
-    fn default_suites_are_fips() {
-        assert!(super::DEFAULT_CIPHER_SUITES
-            .iter()
-            .all(|scs| scs.fips()));
-    }
+    // NOW REMOVED FROM THIS FORK:
+    // #[cfg(... "fips")]
+    // #[test]
+    // fn default_suites_are_fips() {
+    //     ...
+    // }
```

Comment:
```suggestion
    // fn default_suites_are_fips() ...
```

I don't really like this but think this would be best for consistency with some other updates and be more compact.